lib/format-url.js

```diff
@@ -5,9 +5,18 @@ export default function (url) {
     return '';
   }
 
-  if (startsWith(url, '//')) {
-    return 'https:' + url;
+  let t = String(url).trim();
+  if (!t) {
+    return '';
+  }
+
+  if (startsWith(t, '//')) {
+    t = 'https:' + t;
+  }
+
+  if (!startsWith(t, 'http://') && !startsWith(t, 'https://')) {
+    t = 'http://' + t;
   }
 
-  return encodeURI(url);
+  return encodeURI(t);
```

Comment:
hm... I wonder if we should do this here or in html-to-article-json or so... this normalizing kinda looks general.